value-processor/src/org/immutables/value/processor/meta/ValueAttribute.java

```diff
@@ -581,6 +581,10 @@ public boolean isGenerateTransientDerived() {
         && !containingType.serial.isSimple();
   }
 
+  public boolean isGenerateDerived() {
```

Comment:
the getter is not needed here, field works just fine

Reply:
I must have had something else wrong when I first modified the `Immutables.generator` file.  But after rebuilding, it appears I didn't need to do that.

Reply:
sorry, that crazy mix of fields and accessors grew "organically", just don't want to change some aspects of it (as trying to remember which ones computed up-front, which lazily and where). thank you for understanding